### PR TITLE
[FUSEQE-4299] fix databucket tests with new react UI

### PR DESCRIPTION
**Pull request by wyvie**
This fixes databucket tests for the new react UI.
The only test that doesn't work yet, is `@data-buckets-usage`, due to datamapper problems (https://github.com/syndesisio/syndesis/issues/5622).
When datamapper bug gets fixed, all databucket tests should pass.

//test: `@data-buckets-check-popups or @data-buckets-usage-2`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/
  DataMapper.java:7: Wrong order for 'io.syndesis.qe.pages.SyndesisPageObject' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/
  DataMapper.java:9: 'lombok.extern.slf4j.Slf4j' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/
  DataMapper.java:10: Wrong order for 'org.openqa.selenium.By' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/
  DataMapper.java:17: Wrong order for 'com.codeborne.selenide.CollectionCondition.size' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/
  IntegrationFlowView.java:18: Wrong order for 'io.syndesis.qe.pages.SyndesisPageObject' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/
  IntegrationFlowView.java:20: 'lombok.extern.slf4j.Slf4j' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/
  IntegrationFlowView.java:59:32: Name 'Expand' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [ConstantName]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/
  IntegrationFlowView.java:60:32: Name 'Collapse' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [ConstantName]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/
  IntegrationFlowView.java:145:42: String literal expressions should be on the left side of an equalsIgnoreCase comparison. [EqualsAvoidNull]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:8:8: Unused import - cucumber.api.PendingException. [UnusedImports]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:9: Wrong order for 'org.assertj.core.api.Condition' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:26: Wrong order for 'io.syndesis.qe.pages.ModalDialogPage' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:37: 'lombok.extern.slf4j.Slf4j' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:88: 'lambda arguments' has incorrect indentation level 16, expected level should be 12. [Indentation]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:89: 'lambda arguments' has incorrect indentation level 16, expected level should be 12. [Indentation]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:105: 'lambda arguments' has incorrect indentation level 16, expected level should be 12. [Indentation]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:106: 'lambda arguments' has incorrect indentation level 16, expected level should be 12. [Indentation]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:117: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:127: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:132: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:139: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:145: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:151: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:159: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:166: @And should not be used. [RegexpSingleline]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:174:40: String literal expressions should be on the left side of an equalsIgnoreCase comparison. [EqualsAvoidNull]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:176:47: String literal expressions should be on the left side of an equalsIgnoreCase comparison. [EqualsAvoidNull]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:306: 'lambda arguments' has incorrect indentation level 32, expected level should be 28. [Indentation]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/
  IntegrationSteps.java:33:8: Unused import - cucumber.api.java.en.And. [UnusedImports]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert
Connection - CRUD | Credentials | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Integration - Databucket | Create extended
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by wyvie** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert
Integration - Databucket | Create extended
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert
Integration - Databucket | Create extended
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by wyvie** [comment]
So i've found out this fails because of this: https://github.com/syndesisio/syndesis/issues/5528
As soon as that issue is fixed, these tests will pass.

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationFlowView.java [hunk 1]
```diff
@@ -31,28 +32,33 @@
...
+            By.cssSelector("a[data-testid=\"integration-editor-step-adder-add-step-before-connection-link\"]");
 
         public static final By FLOW_TITLE = By.cssSelector(".step.start .step-name");
 
         public static final By TRASH = By.className("fa-trash");
 
         public static final By DELETE_BUTTON = By.cssSelector(".modal-footer .btn-danger");
 
+        public static final By STEP =
+            By.cssSelector("div.list-group-item");
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationFlowView.java]: Is the new row necessary?

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**